client/my-sites/themes/themes-selection.jsx

```diff
@@ -135,15 +138,8 @@ class ThemesSelection extends Component {
 			);
 		}
 
-		const options = this.getOptions(
-			themeId,
-			variation,
-			`style variation: ${ variation ? variation.slug : 'show more' }`
-		);
-
-		if ( options && options.preview ) {
-			options.preview.action( themeId );
-		}
+		this.props.setThemePreviewOptions( themeId, null, null, variation );
+		pageRouter( this.props.getThemeDetailsUrl( themeId ) );
```

Comment:
Could we support this feature via the URL? If so, we're able to see the same style variation when we refresh the page. What do you think?

Reply:
Good idea 👍